src/python/visclaw/frametools.py

```diff
@@ -311,14 +386,22 @@ def plot_frame(framesolns,plotdata,frameno=0,verbose=False):
     if verbose:
         print '    Done with plotframe for frame %i at time %g' % (frameno,t)
 
-    
+
     # print the figure(s) to file(s) if requested:
     if (plotdata.mode() != 'iplotclaw') & plotdata.printfigs:
         # iterate over all figures that are to be printed:
-        for figno in plotted_fignos:
-            printfig(frameno=frameno, figno=figno, \
-                    format=plotdata.print_format, plotdir=plotdata.plotdir,\
-                    verbose=verbose)
+        for figname in plotdata._fignames:
+            plotfigure = plotdata.plotfigure_dict[figname]
+            figno = plotfigure.figno
+            if figno in plotted_fignos:
+                if figno in kml_fignos:
+                    printfig(frameno=frameno, figno=figno, \
+                             format=plotdata.print_format, plotdir=plotdata.plotdir,\
+                             verbose=verbose,kml_fig=True,kml_dpi=plotfigure.kml_dpi)
```

Comment:
But would this then mean that all figures have the same dpi?  I think 200 is probably fine for figures that will be visualized using the web interface.   But for GE, the figures are tiled, and so a much higher dpi is required - 1600?   Since these figures are slower to generate, I wouldn't impose this dpi on all other figures.